tests/scripts/thread-cert/node.py

```diff
@@ -3265,6 +3265,8 @@ def __init__(self, nodeid, name=None, **kwargs):
         self.nodeid = nodeid
         self.name = name or ('Host%d' % nodeid)
         super().__init__(nodeid, **kwargs)
+        self.bash('service otbr-agent stop')
```

Comment:
I wonder if we should also stop mDNSResponder/Avahi in Host to further avoid potential interference.

Reply:
I think it's valid to have an mDNS service running in Host.

Reply:
mDNS service is not useful on Host. Just thinking it makes the test environment more clean to turn it off.

Reply:
I decided to keep mDNS daemons on since `avahi-browse` depends on `avahi-daemon`.